cookbook/network_from_orion_fepp.ipynb

```diff
@@ -15,30 +15,7 @@
    "source": [
```

Comment:
<p>why are we loading the same thing twice?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/275/discussion/#comment-2608491016'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>because I updated how we were doing it and forgot to delete the old way - thanks!</p><p> </p><p>I think this formatting is just a bit clearer.</p><p> </p>